tidy3d/components/data/monitor_data.py

```diff
@@ -575,6 +578,284 @@ class FluxTimeData(MonitorData):
+        )[None, None, :]
+        e_phi = 1j * (l_theta - ETA_0 * n_phi) / 2.0 / self.wavelength(
+            self.frequencies
+        )[None, None, :]
+
+        return DiffractionDataArray(
+            np.stack([e_theta, e_phi], axis=2), coords=self._make_coords_for_pol(["theta", "phi"]))
+
+    @property
+    def H_sph(self) -> DiffractionDataArray:
+        """Return the far field magnetic field in spherical coordinates."""
+        e_theta = self.E_sph.sel(polarization="theta").values
+        e_phi = self.E_sph.sel(polarization="phi").values
+
+        h_theta = -e_phi / ETA_0
+        h_phi = e_theta / ETA_0
+
+        return DiffractionDataArray(
+            np.stack([h_theta, h_phi], axis=2), coords=self._make_coords_for_pol(["theta", "phi"]))
+
+    @property
+    def E_car(self) -> DiffractionDataArray:
+        """Return the far field electric field in Cartesian coordinates."""
+        return self.sph_2_car(self.E_sph)
+
+    @property
+    def H_car(self) -> DiffractionDataArray:
+        """Return the far field magnetic field in Cartesian coordinates."""
+        return self.sph_2_car(self.H_sph)
+
+    @property
+    def amps(self) -> DiffractionDataArray:
+        """Return normalized electric field amplitudes in each order for 's' and 'p' polarizations.
+        Fields are normalized so that the power equals ``abs(amps)^2`` for each polarization.
+        """
+        e_theta = self.E_sph.sel(polarization="theta").values * self.normalization[None, None, :]
+        e_phi = self.E_sph.sel(polarization="phi").values * self.normalization[None, None, :]
+
+        # stack the fields in s- and p-components along a new polarization axis
+        return DiffractionDataArray(
+            np.stack([e_phi, e_theta], axis=2), coords=self._make_coords_for_pol(["s", "p"])
+        )
+
+    # @property
+    # def power(self) -> xr.DataArray:
+    #     """Return the total power in each order, summed over both polarizations."""
+    #     e_x = self.E_car.sel(polarization="x").values
+    #     e_y = self.E_car.sel(polarization="y").values
+
+    #     h_x = self.H_car.sel(polarization="x").values
+    #     h_y = self.H_car.sel(polarization="y").values
+
+    #     # Poynting vector component along z
+    #     s_z = np.abs(0.5 * np.real(
+    #         e_x * np.conj(h_y) - e_y * np.conj(h_x)
+    #     )) * self.sim_size[0] * self.sim_size[1]
+
+    #     coords = self._make_coords_for_pol(["", ""])
+    #     del coords["polarization"]
+    #     s_z_data = xr.DataArray(s_z, coords=coords)
+    #     return s_z_data
+
+    @property
+    def power(self) -> xr.DataArray:
+        """Return the total power in each order, summed over both polarizations."""
+        amps = self.amps
+        angles = self.angles
+        cos_theta = np.cos(np.nan_to_num(angles[0]))
+
+        amps_s = amps.sel(polarization="s")
+        amps_p = amps.sel(polarization="p")
+        return (np.abs(amps_s) ** 2 + np.abs(amps_p) ** 2) * cos_theta
```

Comment:
These three lines could just be `return xr.sum(np.abs(amps)**2, dim="polarization") * cos_theta`.

More importantly, I think the `cos_theta` factor should be already incorporated in `amps` (as a `np.sqrt(cos_theta)`), such that what is written in the docstring is truly correct. Actually I would also reword the docstring of `amps` a little, something like

```
Complex power amplitude in each order for 's' and 'p' polarizations, normalized so that the power carried by the wave of that order and polarization equals ``abs(amps)^2``.
```

I think that the power (amplitude) really should have the cosine factor. Without the cosine factor, what we are returning instead is power density per solid angle (a delta function selecting theta and phi needs to also be added then). The actual power carried in that order is obtained by integrating the power density over a solid angle that includes only that order, and that gives the cos(theta) factor. So let's just use actual power and not power per angle in both `amps` and `power`.

`self.E_sph` on the other hand can be left as is.